articles/openshift/openshift-faq.md

```diff
@@ -85,3 +85,88 @@ Yes. You can use OSBA with Azure Red Hat OpenShift. See [Open Service Broker for
+
++ Cloud provider-initiated maintenance includes network, storage, and regional outages. The maintenance is dependent on the cloud provider and relies on provider-supplied updates.
+
+## General upgrade process
+
+General upgrade process:
+
+
+Running an upgrade should be a safe process to run and should not disrupt cluster services. The SRE can trigger the upgrade process when new versions are available or CVEs are oustanding.
+
+
+Available updates are tested in a stage environment and then applied to production clusters. When applied, a new node is temporarily added and nodes are updated in a rotating fashion so that pods maintain replica counts. Following best practices helps ensure minimal to no downtime.
+
+
+Depending on the severity of the pending upgrade or update, the process might differ in that the updates might be applied quickly to mitigate the service’s exposure to a CVE. A new image will be built asynchronously, tested, and rolled out as a cluster upgrade. Other than that, there is no difference between emergency and planned maintenance. Planned maintenance is not prescheduled with the customer.
+
+Notifications may be sent via ICM and email if communication to the customer is required.
+
+## What about emergency vs. planned maintenance windows?
+
+We do not distinguish between the two types of maintenance. Our teams are available 24/7/365 and do not use traditional scheduled “out-of-hours” maintenance windows.
+
+## How will host operating system and OpenShift software be updated?
+
+The host operating system and OpenShift software is updated through our general upgrade and image build process.
+
+## What’s the process to reboot the updated node?
+
+This should be handled as a part of an upgrade.
+
+## Is data stored in etcd encrypted on ARO?
+
+It is not encrypted on the etcd level. The option to turn it on is currently unsupported. OpenShift supports this feature, but engineering efforts are required to make it on the road map. The data is encrypted at the disk level. Refer to [Encrypting Data at Datastore Layer](https://docs.openshift.com/container-platform/3.11/admin_guide/encrypting_data.html) for more information.
+
+## Can logs of underlying VMs be streamed out to a customer log analysis system?
+
+Syslog, docker logs, journal, and dmesg are handled by the managed service and are not exposed to customers.
+
+## How can a customer get access to metrics like CPU/memory at the node level to take action to scale, debug issues, etc. I cannot seem to run `kubectl top` on an ARO cluster.
+
+`kubectl top` is not available on Red Hat OpenShift. It requires a backing metrics source, either Heapster (deprecated) or metrics-server (incubating/alpha), neither of which are included in the OpenShift monitoring stack.
```

Comment:
try not to put words with / separating them. This causes issues with translation

Reply:
Will separate.